packages/zod-openapi/test/index.test.ts

```diff
@@ -1270,7 +1279,9 @@ describe('Named params in nested routes', () => {
   it('Should return a correct path', async () => {
     const res = await root.request('/doc')
     expect(res.status).toBe(200)
-    const data = await res.json()
+    const data: {
```

Comment:
I think this change is not necessary. What is it for?

Reply:
We can ignore the typescript error, so please remove it!

Reply:
Ok. Removed.